React/Views/RCTModalHostViewManager.m

```diff
@@ -49,17 +51,48 @@ @interface RCTModalHostViewManager () <RCTModalHostViewInteractor>
 
 @end
 
+#if !TARGET_OS_TV
+static UIInterfaceOrientationMask _appSupportedOrientationMask;
+#endif
+
 @implementation RCTModalHostViewManager
 {
   NSHashTable *_hostViews;
 }
 
+#if !TARGET_OS_TV
```

Comment:
Why should we have it in `View manager` (view managers usually are about RN-specific stuff), why not move this to `RCTModalHostView`?